qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -2326,7 +2326,6 @@ class Kubernetes implements OrchestratorMain {
         Container container = new Container(
                 name: deployment.containerName ? deployment.containerName : deployment.name,
                 image: deployment.image,
-                imagePullPolicy: "IfNotPresent",
```

Comment:
This was added with #11829. I would argue that there is no benefit to changing the default unless we know that image prefetch is in use.